SU-SWS/saa_victoria_subtheme
# Adapt 526  style requests

# READY FOR REVIEW

# Summary
- Fixed style changes Michelle requested

# Review By (Date)
- 7/28

# Urgency
- medium

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. Check these items:

- The red banner paragraph size seems small compared to the rest of the content? Can we bump it up to maybe the paragraph size? 22.5px / modular scale 1
- The icon on the icon card seems very small. Maybe bumping it to 48 
-  I'd like to remove the underline on action links but keep the underline on links in paragraph formats including adding the underline to the paragraph on the icon card. Note the paragraph has a regular link and then there is a CTA action link after the paragraph. CTA links - no underline on regular or hover, and regular/in-text links should have an underline always (regular AND on hover).

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-526 https://stanfordits.atlassian.net/browse/ADAPT-526
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
